learn/what_is_meilisearch/comparison_to_alternatives.mdx

```diff
@@ -235,7 +237,7 @@ Its ability to run on a few MBs of RAM makes it a minimalist and resource-effici
 
 #### Typesense
 
-Like Meilisearch, Typesense is a lightweight open-source search engine optimized for speed. We are currently re-evaluating its features and functionality to better understand how it compares with Meilisearch.
+Like Meilisearch, Typesense is a lightweight open-source search engine optimized for speed. To better understand how it compares with Meilisearch, refer to our detailed [Typesense comparison page](https://blog.meilisearch.com/meilisearch-vs-typesense/).
```

Comment:
Can you please add `?utm_campaign=oss&utm_source=docs&utm_medium=comparison` at the end of the URL?